Plugins/Flow.Launcher.Plugin.Program/Main.cs

```diff
@@ -77,13 +78,15 @@ public List<Result> Query(Query query)
                 uwps = _uwps;
             }
 
+            var searchText = WordsHelper.HasChinese(query.Search) ? WordsHelper.GetPinyin(query.Search) : query.Search;
```

Comment:
This should be part of Translate so other plugins could benefit from as well right?

Reply:
As mentioned in my last response, this is for performance. Maybe we can move it to a more general place to translate query if using pinyin so that other plugin can benefit from it, too.

Reply:
Please see my comment above. I am sure users also use Pinyin search for control panel and other plugins right?

Reply:
Actually whether translate the query won't affect people search with pinyin, but will affect people search with raw Chinese character if pinyin is enabled. As my last post has suggested, I would like to put it to somewhere the query is created.